js/support.validate.js

```diff
@@ -0,0 +1,138 @@
+var constraints = {
```

Comment:
Is this code utilizing a standardized validate.js file?   
There is no need for a custom file. This still doesn't go towards the goal we have asked.  We do not like custom code.
Custom Code is expensive to update & maintain.
Why can this not use a standard validate.js file that is forked from a popular repo online so we do not have to constantly ask Devs to keep it up to date?

We never want a scenario where we are created custom code where a template would've done just fine.
https://github.com/search?q=validate.js
Using this link, you can see a few popular validate.js options. There might be something even more standardized in the market.

Then, after we utilize our main validate.js file, we can put an "include" in it to reference a "custom.validate.js" file if we want.
Inside this custom file can contain anything specific for this Repo.

In this fashion, we can duplicate the main validate.js file across all our Repos (we have over 80) and then only update the custom.validate.js file as needed for each site.

This is a cleaner way to maintain our code and how to handle over 80 Repos.

Reply:
Yes we are using validate.js in the code.
But we also need to show error and handle form submit to do that we need custom js file.
https://validatejs.org/examples.html
for example, right now we need to verify Recaptcha is checked or not. Then, we are sending the request through fetch and we need sweet alert in order to open a popup and show an error or success to the user. Also, the styling of all the forms is not the same.
I've used the standard example from validate.js

Reply:
My end goal can still be achieved.

- a master validate.js file that is duplicated across every single Repo
- a custom.validate.js file that compliments the master validate file - to compensate for custom styling, custom form fields, etc

In this fashion, we can update the master as needed when cyber security issues are detected or the industry shifts in new directions.
Then, we can update the custom form as needed.
This keeps all future Devs streamlined, too.

Your work should be turned into a template that we can then copy/paste onto all future repos.

The Sweet Alert addition can be put into an included file instead of directly in the .html file, too. 
We want to keep all the bloat out of the .html file & use includes instead as much as possible.

Reply:
Hi @kinshark Can we change the content of custom validate.js in each repo or it will remain the same across all repos. If you want it same across all repos then how will you handle custom styling of errors and form submission?

Reply:
The validate.js file should include all the basics that are common across all Repos.
Then, the custom.validate.js can have the custom styling, custom errors, etc.

This means we include both files on each page that has a form.

Reply:
And what we will do if a repo has two or more then what we will do with custom validate.js?

Reply:
name the custom validation file in regards to the .html file it references if you need 2 completely different custom files.
only make a secondary custom file if you are duplicating code or it is getting messy.
Otherwise, keep each custom area inside the same custom.validation.js file.

Reply:
for example a Repo with the following:

- support.html
- contactus.html
- feedback.html

Could have:
- js/validate.js
- js/custom.support.validate.js
- js/custom.contactus.validate.js
- js/custom.feedback.validate.js
 

------------
or, it could have:
js/validate.js
js/custom.validate.js
with the following areas inside of custom: 
- support section
- contact us section
- feedback section

Use either method that you find most appropriate.